arangopipe/arangopipe/arangopipe_storage/arangopipe_api.py

```diff
@@ -46,10 +47,10 @@ class ArangoPipe:
     (4) Register you model with ArangoPipe
     """
 
-    def __init__(self, config: ArangoPipeConfig):
+    def __init__(self, db: StandardDatabase, config: ArangoPipeConfig):
         self.cfg = config.get_cfg()
```

Comment:
As far as I can tell, with the database now being passed to the `__init__` method here, the only remaining part of the configuration that is still being used - is the enterprise graph name on line 272. We could decide to either:
- Accept the graph name as a string with a default constant value? 
- Or still accept the ArangoPipeConfig object but only store the graph name?
What do you think?

Reply:
@geenen124, is there anything else that we may put through config in the future? If yes, leaving the `ArangoPipeConfig` with graph name only for now is the best option in my view. Let's discuss on the sync. :-)